packages/core/src/routes-me/user.ts

```diff
@@ -133,7 +133,7 @@ export default function userRoutes<T extends AuthedMeRouter>(
       const { id: userId } = ctx.auth;
       const { password } = ctx.guard.body;
 
-      const { isSuspended } = await findUserById(userId);
+      const { isSuspended, passwordEncrypted: oldPasswordEncrypted } = await findUserById(userId);
```

Comment:
this can be covered in integration tests

Reply:
For sure